prac_07/project_management.py

```diff
@@ -0,0 +1,130 @@
+import datetime
+
+
+class Project:
+    def __init__(self, name, start_date, priority, estimate, completion):
+        self.name = name
+        self.start_date = start_date
+        self.priority = priority
+        self.estimate = estimate
+        self.completion = completion
+
+    def __lt__(self, other):
+        """Comparison based on priority ."""
+        return self.priority < other.priority
+
+    def __str__(self):
+        return f"{self.name}, start: {self.start_date}, priority {self.priority}, estimate: ${self.estimate:.2f}, " \
+               f"completion: {self.completion}%"
```

Comment:
All of this code is your project.py when you refrenece it later in your project management it will allow you to import the class Project so you wont have to have this code in project management but instead just import from project